src/lib/response-triggers/remove.ts

```diff
@@ -0,0 +1,88 @@
+import Trigger from '../../models/trigger';
+import Config from '../config-interface';
+import { DbUsers, DbLocation } from '../database/db-interfaces';
+import { MessageObject } from 'webex/env';
+import { Role, UserRow } from '../../models/database';
+import { Command } from '../../models/command';
+import constants from '../../constants';
+import xmlMessageParser from '../parsers/xmlMessageParser';
+import { Logger } from '../logger';
+
+export default class Remove extends Trigger {
```

Comment:
RemoveUserCommand? Or something like that?

Reply:
Done